src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -1428,6 +1359,106 @@ const formConfig = {
+                      formData?.applicantName?.first
+                    } ${formData?.applicantName?.last} and their new spouse`,
+                ),
+                ...applicantMarriageCertConfig.uiSchema,
+                applicantSecondMarriageCert: fileUploadUI(
+                  "Upload the applicant's second marriage certificate",
+                  fileWithTextConfigBoilerPlate,
+                ),
+              },
+            },
+          },
+          schema: applicantListSchema([], {
+            titleSchema,
+            ...applicantMarriageCertConfig.schema,
+            applicantSecondMarriageCert: fileWithMetadataSchema(
+              acceptableFiles.spouseCert,
+            ),
+          }),
+        },
+        // If applicant remarried after 55 but the second marriage is not viable,
+        // upload a certificate proving the marriage dissolved
+        page18f8: {
+          path: 'applicant-information/:index/second-marriage-dissolved',
+          arrayPath: 'applicants',
+          showPagePerItem: true,
+          title: item =>
+            `${applicantWording(item)} second marriage dissolution documents`,
+          depends: (formData, index) => {
+            if (index === undefined) return true;
+            return (
+              get(
+                'applicantRelationshipToSponsor.relationshipToVeteran',
+                formData?.applicants?.[index],
+              ) === 'spouse' &&
+              get(
+                'applicantSponsorMarriageDetails.relationshipToVeteran',
+                formData?.applicants?.[index],
+              ) === 'marriedTillDeathRemarriedAfter55' &&
+              !get('remarriageIsViable', formData?.applicants?.[index])
+            );
+          },
+          CustomPage: FileFieldCustom,
+          CustomPageReview: AppMarriageDocReviewField,
+          customPageUsesPagePerItemData: true,
+          uiSchema: {
+            applicants: {
+              'ui:options': { viewField: ApplicantField },
+              items: {
+                ...titleUI(
+                  'Optional supporting file upload',
+                  ({ formData }) =>
+                    `Upload the legal document that ended ${
+                      formData?.applicantName?.first
+                    } ${formData?.applicantName?.last}'s second marriage`,
+                ),
+                ...applicantSecondMarriageDivorceCertConfig.uiSchema,
+                applicantSecondMarriageDivorceCert: fileUploadUI(
+                  "Upload the applicant's second marriage dissolution document",
+                  fileWithTextConfigBoilerPlate,
+                ),
+              },
+            },
+          },
+          schema: applicantListSchema([], {
+            titleSchema,
+            ...applicantSecondMarriageDivorceCertConfig.schema,
+            applicantSecondMarriageDivorceCert: fileWithMetadataSchema(
+              acceptableFiles.spouseCert,
+            ),
+          }),
+        },
```

Comment:
This is the main event - adding two file upload pages with custom wording and specific `depends` functionality:

First page:
- If the applicant was married to the sponsor up until time of death AND remarried after age 55 if sponsor died, OR if is currently married to the living sponsor, collect marriage proof

Second Page:
- If sponsor passed and applicant remarried after age 55 AND the remarriage has since ended, collect documents of dissolution